src/renate/updaters/experimental/offline_er.py

```diff
@@ -83,6 +84,21 @@ def train_dataloader(self) -> DataLoader:
                 pin_memory=True,
                 collate_fn=self._train_collate_fn,
             )
+        else:
+            # Manually create a dataloader for the current task with total batch size.
```

Comment:
Can we reuse the code instead of overriding it?

```
self._batch_size += self._memory_batch_size
loaders["current_task"] = super().train_dataloader()
```

Reply:
I can do something like that if you prefer it. We would have to undo the change after creating the data loader though.